vm/proxyapp/proxyrpc/proxyrpc.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2022 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package proxyrpc
+
+import (
+	"time"
+)
+
+type CreatePoolParams struct {
+	Debug bool
+	Param string
+}
+
+type CreatePoolResult struct {
+	Count int // signal the created pool size
+}
+
+type CreateInstanceParams struct {
+	Workdir string
+	Index   int
+}
+
+type CreateInstanceResult struct {
+	ID string // allocated instance id
+}
+
+type CopyParams struct {
+	ID      string
+	HostSrc string
+}
+
+type CopyResult struct {
+	VMFileName string
+}
+
+type ForwardParams struct {
+	ID   string
+	Port int
+}
+
+type ForwardResult struct {
+	ManagerAddress string
+}
+
+type RunStartParams struct {
+	ID      string
+	Timeout time.Duration
```

Comment:
This should be unused now since we handle timeout in vm/proxyappclient.go itself.